data/avgi/avgi side missions.txt

```diff
@@ -316,7 +316,7 @@ mission "Avgi Culture: Park Politics"
 				`	"What about the Dissonance? I've heard of them."`
 					goto dissonance
 
-			`	"Contemplation. Perhaps a better question would be, 'what it was like before the hyperdrive.' For our society was not placed into turmoil first by the Aberrant, but rather by the discovery of a means to travel faster than light. Inside a handful of years an outlandish physics experiment went from theory to reality, and stars that had been decades or centuries away instead became reachable in first months, then weeks, then days as the hyperdrive continued to improve."`
+			`	"Contemplation. Perhaps a better question would be, 'what it was like before the hyperdrive?' For our society was not placed into turmoil first by the Aberrant, but rather by the discovery of a means to travel faster than light. Inside a handful of years an outlandish physics experiment went from theory to reality, and stars that had been decades or centuries away instead became reachable in first months, then weeks, then days as the hyperdrive continued to improve."`
```

Comment:
```suggestion
			`	"Contemplation. Perhaps a better question would be, 'what it was like before the hyperdrive?' For our society was not placed into turmoil first by the Aberrant, but rather by the discovery of a means to travel faster than light. Inside a handful of years an outlandish physics experiment went from theory to reality, and stars that had been decades or centuries away instead became reachable in first months, then weeks, then days as the hyperdrive continued to improve.`
```

Reply:
It's a rule; if there are multiple paragraphs of dialog by the same person all except for the last paragraph should lack the ending quotation mark.